cgpclient/scripts/upload_dragen_fastq_list.py

```diff
@@ -47,24 +54,30 @@ def parse_args(args: list[str]) -> argparse.Namespace:
         required=True,
     )
     parser.add_argument(
-        "-r",
-        "--ngis_referral_id",
+        "-i",
+        "--run_id",
         type=str,
-        help="NGIS referral identifier for the sample",
+        help="Unique identifier for the sequencing run that generated the FASTQs",
```

Comment:
possibly add an "e.g. for an Illumina sequencer this should be the run folder name such as 240627_M03456_0001_AHCYL3XY "